cli/zone.go

```diff
@@ -40,20 +68,30 @@ Fetches and displays the zone configuration for <key-prefix>.
 	Run: runGetZone,
 }
 
-// runGetZone invokes the REST API with GET action and key prefix as path.
+// runGetZone retrieves the zone config for a given object id,
+// and if present, outputs its YAML representation.
 func runGetZone(cmd *cobra.Command, args []string) {
 	if len(args) != 1 {
 		cmd.Usage()
 		return
 	}
-	admin := client.NewAdminClient(&context.Context, context.Addr, client.Zone)
-	body, err := admin.GetYAML(args[0])
+	id, err := strconv.Atoi(args[0])
```

Comment:
I know this is on your todo list, but add a `TODO` that this should be taking a table/database name is input and converting that to an ID.